OpenEdXMobile/src/main/java/org/edx/mobile/social/SocialLoginDelegate.java

```diff
@@ -227,7 +225,7 @@ private SocialButtonClickHandler(SocialFactory.SOCIAL_SOURCE_TYPE socialType) {
         @Override
         public void onClick(View v) {
             if (!NetworkUtil.isConnected(activity)) {
-                callback.showErrorDialog(activity.getString(R.string.no_connectivity),
+                callback.showErrorMessage(activity.getString(R.string.no_connectivity),
                         activity.getString(R.string.network_not_connected));
```

Comment:
The callback interface defined only those events that are already implemented in the (base) Activity, so you shouldn't customize it. Just change it back to `showErrorDialog()`, and we should be fine.